src/services/k8s/knative.go

```diff
@@ -105,6 +164,13 @@ func Apply(namespace string, config *rest.Config, project *project.Project, dock
 		return fmt.Errorf("cannot create namespace %s, failed with %v", serviceManifest.ObjectMeta.Namespace, err)
 	}
 
+	envVarList, err := loadEnvFile(envFile)
+	if err != nil {
+		return err
+	}
+
+	serviceManifest.Spec.Template.Spec.Containers[0].Env = append(serviceManifest.Spec.Template.Spec.Containers[0].Env, envVarList...)
+
```

Comment:
I think it could be beneficial to move this in the `loadManifest` function, by doing this you could also test this better by calling loadManifest with different inputs and testing the result manifest structure.

Reply:
moved to load manifest function. 
in terms of testing I took out couple of cases that were checking load env file functionality & added sample env file as attribute to loadmanifest test case.